Tomas Heran <tomas.heran@oracle.com>
Need to install header files to /usr/include/jansson
This patch will not be reported upstream.

diff -r fa5b56b77f9c -r 10395313a7ae jansson.pc.in
--- a/jansson.pc.in	Tue Oct 28 16:31:32 2014 +0100
+++ b/jansson.pc.in	Tue Oct 28 17:05:35 2014 +0100
@@ -1,7 +1,7 @@
 prefix=@prefix@
 exec_prefix=@exec_prefix@
 libdir=@libdir@
-includedir=${prefix}/include
+includedir=${prefix}/include/jansson
 
 Name: Jansson
 Description: Library for encoding, decoding and manipulating JSON data
